internal/controller/trigger/controller.go

```diff
@@ -119,7 +119,7 @@ func (ctrl *controller) ResetOffsetToTimestamp(ctx context.Context,
 		return nil, errors.ErrResourceNotFound.WithMessage("subscription not exist")
 	}
 	if sub.Phase != metadata.SubscriptionPhaseRunning {
-		return nil, errors.ErrResourceCanNotOp.WithMessage("subscription is not running")
+		return nil, errors.ErrServerNotRunning.WithMessage("subscription is not running")
```

Comment:
What is the mean of `ErrServerNotRunning`?

Reply:
I think both “not ready” and "no server" can be described as ServerNotRunning. Unless this error needs to be handled separately by the upper layer, it is unnecessary to define a separate error code, and you can see the specific cause of the error msg